### ISSUE TITLE
I have doubts about the analysis results with 6x6

### ISSUE THREAD

**Issue by hope366** [lightvector/KataGo#868]
katago-1.13.0-cuda
kata1-b18c384nbt-s7709731328-d3715293823 
japanese-rule

At Black 17, Black is +35. katago is almost correct.
![11無題](https://github.com/lightvector/KataGo/assets/63999713/1cc86121-0eb3-4e57-9a44-c4220616dbdb)

White 18 has no effect on winning or losing, but strictly speaking it is a one-move pass and a one-point loss. Therefore, in this diagram, black is +36. However, katago is showing 33-34.
![1無題](https://github.com/lightvector/KataGo/assets/63999713/e93a4100-1fab-45cb-8ce7-cbe6d9437ddd)

katago-1.13.0-cuda
Finetuned 9x9 Neural Net(kata9x9-b18c384nbt-20231025.bin.gz )
japanese-rule

We analyzed the same situation with the recently released 9x9 net.
I got a strange value of black +46 to 47.
![無題](https://github.com/lightvector/KataGo/assets/63999713/2cb18f63-8260-4cf8-86e8-31e662da20b5)

**Comment by lightvector** [comment]
Thanks! Yes, everything you're reporting is expected, there isn't anything wrong.

* The 9x9 finetuned neural net is for 9x9 only. That's why it was advertised as a 9x9-finetuned neural net. It no longer has much experience with understanding how different board sizes behave. So it is completely normal if it gives worse evaluations on boards that are not 9x9. You should use the normal net instead.

* None of the neural nets are trained on 6x6. So their score estimates are not guaranteed to be reliable on 6x6. However the normal net does have experience with all sizes from 7x7 to 19x19, so it can probably have some extrapolation to 6x6 even if it's not perfect.

* It's well-known that the score estimates produced by MCTS are noisy, because maximizing score is only a secondary objective of the search. This is especially true when the score estimate is very far from 0. In this case, attempting to use the score estimate from MCTS is usually the wrong approach. Try instead adjusting komi and seeing which values of komi result in a game that is a draw, or a win, or a loss.

**Comment by hope366** [comment]
Thank you for your detailed explanation.
I adjusted the value of komi to a value that would result in a tie.
It was komi=32. Originally it was supposed to be a draw with komi=35, but that didn't happen.
![無題](https://github.com/lightvector/KataGo/assets/63999713/c7945d69-de2a-4acd-b9fd-12c53b9fede9)

Next, I made a similar diagram using 7x7. The desired komi value for a draw is 48, but it was actually 47.
![1無題](https://github.com/lightvector/KataGo/assets/63999713/2eda1135-2ece-4854-9ecb-3214cd70001d)
